packages/convex-vue/README.md

```diff
@@ -8,16 +8,19 @@
 npm install @convex-vue/core @vueuse/core convex vue-router
 ```
 
-- create the vue plugin
+### Simple example
```

Comment:
Avoid the word "simply" almost everywhere. It's simple to the author but rarely simple to the reader and doesn't offer additional information: https://blog.knowledgeowl.com/blog/posts/dont-say-simply-jim-fisher/

```suggestion
### Example
```